Generator tests fail locally

I'm getting this failure from the `generator-react-server` tests locally

```
> generator-react-server@0.5.1 test /Users/dwade/foss/react-server/packages/generator-react-server
> npm run ava && npm run lint && nsp check
> generator-react-server@0.5.1 ava /Users/dwade/foss/react-server/packages/generator-react-server
> ava test --tap
TAP version 13
# generator-react-server:app .reactserverrc matches react-server-cli default option keys
ok 1 - generator-react-server:app .reactserverrc matches react-server-cli default option keys
# generator-react-server:app creates default files
ok 2 - generator-react-server:app creates default files
# generator-react-server:app creates docker files
ok 3 - generator-react-server:app creates docker files
# generator-react-server:app passes the test target
not ok 4 - generator-react-server:app passes the test target
  ---
    operator: ===
    expected: true
    actual: false
    at: "/var/folders/36/7y7zl4bx75nb2psx96nm8f600000gn/T/ff34fd10883abd5f43b83c08c6782253d809bb68"
  ...
1..4
# tests 4
# pass 3
# fail 1
NpmUtilities.execInDir        ("run test", [], "./packages/generator-react-server")
> generator-react-server@0.5.1 test /Users/dwade/foss/react-server/packages/generator-react-server
> npm run ava && npm run lint && nsp check
> generator-react-server@0.5.1 ava /Users/dwade/foss/react-server/packages/generator-react-server
> ava test --tap
TAP version 13
# generator-react-server:app .reactserverrc matches react-server-cli default option keys
ok 1 - generator-react-server:app .reactserverrc matches react-server-cli default option keys
# generator-react-server:app creates default files
ok 2 - generator-react-server:app creates default files
# generator-react-server:app creates docker files
ok 3 - generator-react-server:app creates docker files
# generator-react-server:app passes the test target
not ok 4 - generator-react-server:app passes the test target
  ---
    operator: ===
    expected: true
    actual: false
    at: "/var/folders/36/7y7zl4bx75nb2psx96nm8f600000gn/T/ff34fd10883abd5f43b83c08c6782253d809bb68"
  ...
1..4
# tests 4
# pass 3
# fail 1
NpmUtilities.runScriptInDir   ("test", [], "./packages/generator-react-server")
> generator-react-server@0.5.1 test /Users/dwade/foss/react-server/packages/generator-react-server
> npm run ava && npm run lint && nsp check
> generator-react-server@0.5.1 ava /Users/dwade/foss/react-server/packages/generator-react-server
> ava test --tap
TAP version 13
# generator-react-server:app .reactserverrc matches react-server-cli default option keys
ok 1 - generator-react-server:app .reactserverrc matches react-server-cli default option keys
# generator-react-server:app creates default files
ok 2 - generator-react-server:app creates default files
# generator-react-server:app creates docker files
ok 3 - generator-react-server:app creates docker files
# generator-react-server:app passes the test target
not ok 4 - generator-react-server:app passes the test target
  ---
    operator: ===
    expected: true
    actual: false
    at: "/var/folders/36/7y7zl4bx75nb2psx96nm8f600000gn/T/ff34fd10883abd5f43b83c08c6782253d809bb68"
  ...
1..4
# tests 4
# pass 3
# fail 1
Errored while running npm script 'test' in 'generator-react-server'
Error: Command failed: npm run test
{ Error: Command failed: npm test
  1 failed
  server is running
  Error: connect ECONNREFUSED 127.0.0.1:3000
npm ERR! Test failed.  See above for more details.
    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5) killed: false, code: 1, signal: null, cmd: 'npm test' }
> foo@0.0.1 test /private/var/folders/36/7y7zl4bx75nb2psx96nm8f600000gn/T/ff34fd10883abd5f43b83c08c6782253d809bb68
> npm run lint && nsp check && ava test.js
> foo@0.0.1 lint /private/var/folders/36/7y7zl4bx75nb2psx96nm8f600000gn/T/ff34fd10883abd5f43b83c08c6782253d809bb68
> eslint test.js pages
(+) No known vulnerabilities found
  1 failed
  server is running
  Error: connect ECONNREFUSED 127.0.0.1:3000
npm ERR! Test failed.  See above for more details.
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "ava"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! generator-react-server@0.5.1 ava: `ava test --tap`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the generator-react-server@0.5.1 ava script 'ava test --tap'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the generator-react-server package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ava test --tap
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs generator-react-server
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls generator-react-server
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/packages/generator-react-server/npm-debug.log
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "test"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! generator-react-server@0.5.1 test: `npm run ava && npm run lint && nsp check`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the generator-react-server@0.5.1 test script 'npm run ava && npm run lint && nsp check'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the generator-react-server package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run ava && npm run lint && nsp check
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs generator-react-server
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls generator-react-server
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/packages/generator-react-server/npm-debug.log
    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
Errored while running RunCommand.execute
Error: Command failed: npm run test
{ Error: Command failed: npm test
  1 failed
  server is running
  Error: connect ECONNREFUSED 127.0.0.1:3000
npm ERR! Test failed.  See above for more details.
    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5) killed: false, code: 1, signal: null, cmd: 'npm test' }
> foo@0.0.1 test /private/var/folders/36/7y7zl4bx75nb2psx96nm8f600000gn/T/ff34fd10883abd5f43b83c08c6782253d809bb68
> npm run lint && nsp check && ava test.js
> foo@0.0.1 lint /private/var/folders/36/7y7zl4bx75nb2psx96nm8f600000gn/T/ff34fd10883abd5f43b83c08c6782253d809bb68
> eslint test.js pages
(+) No known vulnerabilities found
  1 failed
  server is running
  Error: connect ECONNREFUSED 127.0.0.1:3000
npm ERR! Test failed.  See above for more details.
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "ava"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! generator-react-server@0.5.1 ava: `ava test --tap`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the generator-react-server@0.5.1 ava script 'ava test --tap'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the generator-react-server package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ava test --tap
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs generator-react-server
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls generator-react-server
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/packages/generator-react-server/npm-debug.log
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "test"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! generator-react-server@0.5.1 test: `npm run ava && npm run lint && nsp check`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the generator-react-server@0.5.1 test script 'npm run ava && npm run lint && nsp check'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the generator-react-server package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run ava && npm run lint && nsp check
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs generator-react-server
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls generator-react-server
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/packages/generator-react-server/npm-debug.log
    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
npm ERR! Test failed.  See above for more details.
```

which is weird because they're passing on Travis.  I don't seem to have anything else running (per `» lsof -n -i:3000 | grep LISTEN`), and it starts up outside of the tests, so I don't get it.

--- a/packages/react-server-module-tagger/index.js
+++ b/packages/react-server-module-tagger/index.js
@@ -1,35 +1,6 @@
 var isWindows = ('win32' === process.platform);
 
 /**
- * A util function for tagging modules.
- * @example
- *     tagger({filePath: '/path/to/my/file.js', trim: 'path.to.', opts: {label: "foo"})
- *     // returns '{\"label\":\"foo\",\"name\":\"my.file\",\"color\":{\"server\":87,\"client\":\"rgb(42,212,212)\"}}'
- * @param  {String} filePath  The path to the file
- * @param  {String} opts      The options, including a label, to add to the module tag
- * @param  {String} trim      The prefix to remove from the logger name
- * @param  {String} basePath  The path to the root of the project
- * @param  {String} prefix    A prefix to prepend to the logger name
- * @return {String}           A json object containing a module identifier
- */
-module.exports = function(arg){
-	var opts = arg.opts || {}
-	,   fn   = arg.filePath
-	,   trim = arg.trim || ''
-	,   basePath = arg.basePath || ''
-	,   prefix = arg.prefix
-
-	if (fn.indexOf(basePath) !== 0) {
-		throw new Error("Unable to handle " + basePath + " for " + fn);
-	}
-
-	opts.name  = getName  (fn, opts, trim, basePath, prefix);
-	opts.color = getColor (opts);
-
-	return JSON.stringify(opts);
-}
-
-/**
  * Gets the name of a logger from its filepath.
  * @example
  *     getName(
@@ -119,3 +90,33 @@ var getColor = (function(){
 		return colors[hash(opts.name)];
 	}
 })();
+
+
+/**
+ * A util function for tagging modules.
+ * @example
+ *     tagger({filePath: '/path/to/my/file.js', trim: 'path.to.', opts: {label: "foo"})
+ *     // returns '{\"label\":\"foo\",\"name\":\"my.file\",\"color\":{\"server\":87,\"client\":\"rgb(42,212,212)\"}}'
+ * @param  {String} filePath  The path to the file
+ * @param  {String} opts      The options, including a label, to add to the module tag
+ * @param  {String} trim      The prefix to remove from the logger name
+ * @param  {String} basePath  The path to the root of the project
+ * @param  {String} prefix    A prefix to prepend to the logger name
+ * @return {String}           A json object containing a module identifier
+ */
+module.exports = function(arg){
+	var opts = arg.opts || {}
+		,   fn   = arg.filePath
+		,   trim = arg.trim || ''
+		,   basePath = arg.basePath || ''
+		,   prefix = arg.prefix
+
+	if (fn.indexOf(basePath) !== 0) {
+		throw new Error("Unable to handle " + basePath + " for " + fn);
+	}
+
+	opts.name  = getName  (fn, opts, trim, basePath, prefix);
+	opts.color = getColor (opts);
+
+	return JSON.stringify(opts);
+}

